lua/telescope/builtin/git.lua

```diff
@@ -62,7 +63,7 @@ git.bcommits = function(opts)
     prompt_title = 'Git BCommits',
     finder = finders.new_table {
       results = results,
-      entry_maker = opts.entry_maker or make_entry.gen_from_git_commits(opts),
+      entry_maker = opts.entry_maker or make_entry.gen_from_git_commits(),
```

Comment:
I think it's good to leave `opts` in there in case `gen_from_git_commits` ever needs them.

Just my opinion about it.

Reply:
I am adding them back in makes sense.